Lib/test/test_os.py

```diff
@@ -1816,9 +1816,19 @@ def mock_execve(name, *args):
 class ExecTests(unittest.TestCase):
     @unittest.skipIf(USING_LINUXTHREADS,
                      "avoid triggering a linuxthreads bug: see issue #4970")
+    def test_execv_with_bad_program(self):
```

Comment:
This test is not at the correct place: it took the `@unittest.skipIf()` decorator away from the previous test.

Also, could you add a test for all `os.execv*` functions?

Reply:
> This test is not at the correct place: it took the @unittest.skipIf() decorator away from the previous test.

Good catch. Looking at the details at https://bugs.python.org/issue4970, it looks like all the execv* tests should actually have the same decorator, so I added it to the others.

> Also, could you add a test for all os.execv* functions?

Done.